torch/csrc/autograd/VariableTypeManual.cpp

```diff
@@ -160,7 +160,7 @@ Tensor& requires_grad_(Tensor& self, bool _requires_grad) {
   return self.set_requires_grad(_requires_grad);
 }
 
-void retain_grad(const Tensor & self) {
```

Comment:
Are these kinds of changes BC-breaking for the c++ API? cc @glaringlee

Reply:
I don't think they are? Anywhere you non-const reference you can add const to it.